packages/graph/hash_graph/lib/graph/src/types/mod.rs

```diff
@@ -4,9 +4,27 @@ pub mod schema;
 
 use core::fmt;
 
+use serde::{Deserialize, Serialize};
 use uuid::Uuid;
 
-use crate::types::schema::Uri;
+#[derive(Debug, Clone, PartialEq, Eq, Hash, Serialize, Deserialize, sqlx::Type)]
+#[serde(transparent)]
+#[sqlx(transparent)]
```

Comment:
Crazy to think about how much code we are generating for a wrapper around String 😂

Reply:
https://gist.github.com/thehabbos007/1276fd1acbb73e6f3376c81b5ae093bf it's indeed a lot